test/plugins/test_hover.py

```diff
@@ -21,7 +21,7 @@ def test_hover():
     doc = Document(DOC_URI, DOC)
 
     assert {
-        'contents': 'main()\n\nhello world'
+        'contents': [{'language': 'python', 'value': 'main()'}, 'hello world']
```

Comment:
This could break existing clients.

Reply:
I highly doubt so. If the client doesn't support it -> it doesn't support lsp specification.
https://microsoft.github.io/language-server-protocol/specification#textDocument_hover

this is a format of `MarkedString[]`

Reply:
I think we have to check the text document client capabilities for `hover.contentFormat`, the protocol certainly didn’t support this at some point in the past

Reply:
good point, will check into that

Reply:
> this is a format of MarkedString[]

Right, sorry for the noise.